src/calculateRentalCost.js

```diff
@@ -3,8 +3,20 @@
  *
  * @return {number}
  */
+const carCosts = 40;
+
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * carCosts;
+  }
+
+  if (days >= 3 && days <= 6) {
```

Comment:
Avoid using literals directly in expressions. Instead, use constants to explain these values. For example, define a constant for the discount amount like 'const MEDIUM_TERM_DISCOUNT = 20;' and use it here.